README.md

```diff
@@ -1,49 +1,48 @@
 # Welcome to Rails
 
-Rails is a web-application framework that includes everything needed to
-create database-backed web applications according to the
+Rails as a web-application framework includes everything needed to
+create database-backed web applications with the
 [Model-View-Controller (MVC)](http://en.wikipedia.org/wiki/Model-view-controller)
 pattern.
 
 Understanding the MVC pattern is key to understanding Rails. MVC divides your
-application into three layers, each with a specific responsibility.
+application into three layers: Model, View, and Controller, each with a specific responsibility.
 
-The _Model layer_ represents your domain model (such as Account, Product,
-Person, Post, etc.) and encapsulates the business logic that is specific to
+The _Model layer_ represents the domain model (such as Account, Product,
+Person, Post, etc.) and encapsulates the business logic specific to
 your application. In Rails, database-backed model classes are derived from
-`ActiveRecord::Base`. Active Record allows you to present the data from
+`ActiveRecord::Base`. Active Record allows to present the data from
 database rows as objects and embellish these data objects with business logic
-methods. You can read more about Active Record in its [README](activerecord/README.rdoc).
-Although most Rails models are backed by a database, models can also be ordinary
-Ruby classes, or Ruby classes that implement a set of interfaces as provided by
-the Active Model module. You can read more about Active Model in its [README](activemodel/README.rdoc).
+methods. Read more about Active Record in its [README](activerecord/README.rdoc).
+Although most Rails models are backed by a database, models can be ordinary
+Ruby classes or Ruby classes with a set of interfaces provided by
+the Active Model module. Read more about Active Model in its [README](activemodel/README.rdoc).
 
-The _Controller layer_ is responsible for handling incoming HTTP requests and
-providing a suitable response. Usually this means returning HTML, but Rails controllers
+The _Controller layer_ handles incoming HTTP requests and
```

Comment:
Current version correctly emphasises the contract, other components can be made to step in and handle, but to keep things clean we _want_ to keep this responsibility in the controller layer.